features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/RumContext.kt

```diff
@@ -58,8 +60,12 @@ internal data class RumContext(
         fun fromFeatureContext(featureContext: Map<String, Any?>): RumContext {
             val applicationId = featureContext[APPLICATION_ID] as? String
             val sessionId = featureContext[SESSION_ID] as? String
-            val sessionState =
-                RumSessionScope.State.fromString(featureContext[SESSION_STATE] as? String)
+            val sessionState = RumSessionScope.State.fromString(
+                featureContext[SESSION_STATE] as? String
+            )
+            val sessionStartReason = RumSessionScope.StartReason.fromString(
```

Comment:
we need to update `toMap` method then as well, to write it to the context.

Reply:
Good catch